modular_splurt/code/modules/mob/mob.dm

```diff
@@ -41,3 +41,8 @@
 	if (.)
 		return
 	to_chat(A, "<span class='notice'>[src] seems to be checking you out.</span>")
+
+/mob/proc/is_dumb4cum()
+	if(HAS_TRAIT(src, TRAIT_DUMB4CUM))
+		return TRUE
+	return FALSE
```

Comment:
You can just use HAS_TRAIT() in every check instead of doing a whole different proc for it